docs/docusaurus/docs/oss/changelog.md

```diff
@@ -499,4190 +499,5 @@ slug: /core/changelog
-* [MAINTENANCE] Add docs snippet checker to `dev` CI ([#6594](https://github.com/great-expectations/great_expectations/pull/6594))
-* [MAINTENANCE] Utilize a `StrEnum` for `ConfigPeer` modes ([#6596](https://github.com/great-expectations/great_expectations/pull/6596))
-* [MAINTENANCE] Refactor `BaseDataContext` and `DataContext` into factory functions ([#6531](https://github.com/great-expectations/great_expectations/pull/6531))
-* [MAINTENANCE] flake8 coverage - `render`, `juptyer_ux`, `checkpoint` sub-packages ([#6607](https://github.com/great-expectations/great_expectations/pull/6607))
-* [MAINTENANCE] filter `RemovedInMarshmallow4` warnings ([#6602](https://github.com/great-expectations/great_expectations/pull/6602))
-* [MAINTENANCE] Generate public API candidates ([#6600](https://github.com/great-expectations/great_expectations/pull/6600))
-* [MAINTENANCE] partial `cli` + `usage_stats` typing ([#6335](https://github.com/great-expectations/great_expectations/pull/6335))
-* [MAINTENANCE] update pip installs in pipelines ([#6609](https://github.com/great-expectations/great_expectations/pull/6609))
-* [MAINTENANCE] create cache as part of `--ci` type-checking step ([#6621](https://github.com/great-expectations/great_expectations/pull/6621))
-* [MAINTENANCE] Add `row_condition` logic to `RendererConfiguration` and remove from atomic renderer implementations ([#6616](https://github.com/great-expectations/great_expectations/pull/6616))
-* [MAINTENANCE] install `pydantic` in develop pipeline ([#6624](https://github.com/great-expectations/great_expectations/pull/6624))
-* [MAINTENANCE] Fix develop static type-check stage ([#6628](https://github.com/great-expectations/great_expectations/pull/6628))
-* [MAINTENANCE] Unexpected Counts table in DataDocs able to show counts for sampled values ([#6634](https://github.com/great-expectations/great_expectations/pull/6634))
-* [MAINTENANCE] CI - install from `requirements-types.txt` ([#6639](https://github.com/great-expectations/great_expectations/pull/6639))
-* [MAINTENANCE] Add docstring linter for public api to CI ([#6641](https://github.com/great-expectations/great_expectations/pull/6641))
-* [MAINTENANCE] Batch ID must incorporate batch_spec_passthrough.  Instantiate Validator with DataContext in test modules.  Query metrics/expectations types cleanup. ([#6636](https://github.com/great-expectations/great_expectations/pull/6636))
-* [MAINTENANCE] Skip postgres tests in spark. ([#6643](https://github.com/great-expectations/great_expectations/pull/6643))
-* [MAINTENANCE] Enrich `RendererConfiguration` primitive types ([#6629](https://github.com/great-expectations/great_expectations/pull/6629))
-* [MAINTENANCE] M/great 1225/async builds branch ([#6644](https://github.com/great-expectations/great_expectations/pull/6644))
-* [MAINTENANCE] Comment out calling _disable
```

Comment:
Removed all the `pre` `1.0` changelogs from this doc and fixed this Older Changelist list to point at something that actually exists.

Which makes me think we should do the same thing for the `0.18.x` changelog, only include the `0.18.x` changes and link to the older ones but I don't feel like doing that right now 😆